specification/cognitiveservices/data-plane/VideoSearch/readme.md

```diff
@@ -0,0 +1,32 @@
+# Cognitive Services Video Search SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating Video Search SDK.
```

Comment:
The sections such as 'Configuration' seems to be missing. Please use https://github.com/Azure/azure-rest-api-specs/tree/current/specification/cdn/resource-manager as reference. If you add them only then linter will start processing and the errors, if any, will be reported.

Reply:
The reference should be what we used for EntitySearch and WebSearch, since these are the same family of APIs, right? https://github.com/Azure/azure-rest-api-specs/blob/current/specification/cognitiveservices/data-plane/EntitySearch/readme.md

We do not want an ARM package as far as I know.

Reply:
I understand you do not want an ARM package. But the Readme file must look like:

## Configuration



### Basic Information 
These are the global settings for the Cdn API.

``` yaml
openapi-type: arm
tag: package-2017-04
```

Reply:
This is consistent with all other data-planes. Any reason for it to be different? Also, we have no other information to fill in that configuration section as of now.

Reply:
One of the pain points we've felt during this is that we have different reviewers reviewing our overwhelmingly similar specs. I understand the pipeline your team is building is actively evolving, but a lot of these changes are opaque to us. If there's a new requirement for this configuration section, it should be retroactively applied to all APIs, and communicated better than through the code review of someone creating essentially a copy of a spec that is already approved. 

Can you explain a bit about what this configuration section does? We got linter errors in our current entity search / web search specs (well, we got warnings).

Reply:
@sarangan12  -- It's not an ARM service, the pattern is a bit different; and it looks fine the way it is to me. (tagging with `release_* ` and `openapi-type: data-plane`

Reply:
Thanks folks for the information and help review.